site/content/api/Command-And-Control-API.md

```diff
@@ -12,66 +12,52 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
-| *command* | no | *application-properties* | UTF-8 *string* | MUST contain the command to be executed by a device. |
-|*ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. Default timeout is *60000* (1 minute). |
-|*durable* | no | *properties* | *boolean* | Indicates if the message should be persisted (*store and forward*) by the underlying message transport middleware. By default command&control messages are durable.  |
-
-If a control message requires additional data to be properly executed (like command parameters), the body of the message MUST consist of a single AMQP *Data* section containing a control 
-message data. The format and encoding of the data is indicated by the *command*. It is up to the device to interpret the incoming body by taking `command` into 
-consideration.
-
-## Sending back control message execution result
-
-After execution of a control message on a device, device MUST send a response message indicating a status of that execution.
-
-**Preconditions**
+| *subject*        | yes       | *properties*             | *string*     | MUST contain the command name to be executed by a device. |
+| *correlation-id* | no        | *properties*             | *message-id* | MAY contain an ID used to correlate a response message to the original request. If set, it is used as the *correlation-id* property in the response, otherwise the value of the *message-id* property is used. |
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
 
-1. Client has established an AMQP connection with Hono.
-2. Client has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
-device that received the control message and `${tenant_id}` is the ID of the tenant the device belongs to.
+If a command message requires additional data to be properly executed (like command parameters), the body of the message MUST consist of a single AMQP *Data* section containing a control 
+message data. The format and encoding of the data is indicated by the *subject*. It is up to the device to interpret the incoming body by taking it into 
+consideration.
 
-**Message Flow**
+## Receiving the Command's execution result
 
-*TODO* add sequence diagram
+After execution of a command on a device, the device MUST send a response message indicating a status of that execution.
```

Comment:
... indicating the outcome of executing the command ?

Reply:
OK